src/Peering/Peering/Peering.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>Peering</PsModuleName>
```

Comment:
Please simplify this block to look like this file: https://github.com/Azure/azure-powershell/blob/master/src/DevSpaces/DevSpaces/DevSpaces.csproj